app/services/date_stamper.rb

```diff
@@ -0,0 +1,17 @@
+class DateStamper
+  def initialize(crime_app, case_type)
+    @crime_app = crime_app
+    @case_type = case_type
+  end
+
+  def call
```

Comment:
Could we also have an active record validation on date_stamp presence when case type is date_stampable? ?